src/main/scala/com/globo/bigdata/ghdfs/HdfsManager.scala

```diff
@@ -34,6 +35,19 @@ class HdfsManager(hdfs: FileSystem) {
 
   def write(hadoopPath: Path): FSDataOutputStream = create(hadoopPath: Path)
 
+  def write(hadoopPath: Path, is: InputStream): Boolean = {
+    val fsOutputStream = create(hadoopPath)
```

Comment:
Change the call of the create(..)  to the write(..).

Today create and write are similar, but maybe in the future, we will add new behaviours to the write